Makefile

```
@@ -172,10 +202,19 @@ image-build:
 		$(PUSH) \
 		$(IMAGE_BUILD_EXTRA_OPTS) ./
 
+.PHONY: e2e-docker
+e2e-docker: PLATFORMS=linux/amd64
+e2e-docker: IMAGE_BUILD_EXTRA_OPTS=--load
+e2e-docker: image-build
+
 .PHONY: image-push
 image-push: PUSH=--push
 image-push: image-build
 
+.PHONY: image-load
+image-load:
```

take the following as pseudo-code, if it doesn't work:

```suggestion
kind-image-load: PLATFOMRS=$(KIND_PLATFORM) IMAGE_BUILD_EXTRA_OPTS=--load image-build
```

---

I tried this before but I ran into this problem with docker build buildx: https://github.com/docker/buildx/issues/59

I can't use load and platform together.  It won't push to the kind cluster due to the issue above.  That issue is closed but only because they document that it is not supported.  So I have to run a separate command that builds and loads that for our e2e test.